sdk/build.gradle

```diff
@@ -70,4 +71,18 @@ subprojects {
             include("**/java/**")
         }
     }
+
+    dokkaHtmlPartial {
+        dokkaSourceSets {
+            configureEach {
+                includeNonPublic.set(false)
+                reportUndocumented.set(true)
+                skipDeprecated.set(true)
```

Comment:
nit: I feel like its typical to display documentation for deprecated functions as long as they are appropriately called out as such?

Honestly I say that but this is probably better since if its not in the docs at all, there's less of a chance someone will try to use it.

Reply:
Fair! I think I just copied this from SO. At the moment we don't have deprecated things anyway, but I think we should document them if we ever do.